inc/core/patterns/bcgov-long-card.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * General Long Card block pattern.
+ *
+ * @since 1.0.0
+ *
+ * @package Bcgov/Theme/Block
+ */
+return array(
+	'title'      => __( 'Long Cards', 'bcgov_blocks_theme' ),
+	'categories' => array( 'bcgov-blocks-theme-general' ),
+	'content'    => '<!-- wp:group {"align":"wide","style":{"spacing":{"padding":{"top":"6rem","bottom":"6rem","right":"6rem","left":"6rem"},"blockGap":"3rem"},"border":{"style":"solid","width":"2px"}},"borderColor":"primary-brand-blue"} -->
```

Comment:
I would maybe consider having this on an html page, and then including or requiring this.. ???  as a standard.....